test/app/fiber.test.lua

```diff
@@ -632,30 +632,35 @@ fiber.join(fiber.self())
 sum = 0
 
 -- gh-2694 fiber.top()
-fiber.top_enable()
+is_fiber_top = fiber.top ~= nil
```

Comment:
Nit: It would be nice to drop comment about this workaround for diff-based tests.
For tests in TAP format `fiber.top()`-related test can be skipped. So, IMHO, we should add `FIXME` notice for the comment here.

Reply:
Thanks, fixed.